nexus_workcell_orchestrator/src/workcell_orchestrator.cpp

```diff
@@ -988,8 +978,18 @@ BT::Tree WorkcellOrchestrator::_create_bt(const std::shared_ptr<Context>& ctx)
 {
   // To keep things simple, the task type is used as the key for the behavior tree to use.
   this->_ctx_mgr->set_active_context(ctx);
+  const auto new_task = this->_task_remapper->remap(ctx->task.type);
+  if (new_task != ctx->task.type)
+  {
+    RCLCPP_DEBUG(
+      this->get_logger(),
+      "Loading remapped BT [%s] for original task type [%s]",
+      new_task.c_str(),
+      ctx->task.type.c_str()
+    );
+  }
   return this->_bt_factory->createTreeFromFile(this->_bt_store.get_bt(
-        ctx->task.type));
+        new_task));
```

Comment:
Note here we could either: 
* Use the new name to only load the behavior tree (what I did)
* Override the task type (what I didn't do)

Reply:
I prefer what you did which preserves the original request. Users can always look for the remapping if needed from `ctx-> task_remapper`